nixos/modules/services/x11/desktop-managers/kde5.nix

```diff
@@ -31,204 +31,224 @@ in
         '';
       };
 
+      extraPackages = mkOption {
+        type = types.listOf types.package;
+        default = [];
+        description = ''
+          KDE packages that need to be installed system-wide.
+        '';
+      };
+
     };
 
   };
 
 
-  config = mkIf (xcfg.enable && cfg.enable) {
+  config = mkMerge [
+    (mkIf (cfg.extraPackages != []) {
+      environment.systemPackages = [ (kde5.kdeWrapper cfg.extraPackages) ];
```

Comment:
actually, I meant that the items on https://github.com/abbradar/nixpkgs/blob/3b8fde2b3913fef8a4dffd2eead4121348fcc961/nixos/modules/services/x11/desktop-managers/kde5.nix#L87 would be included, minimizing the amount of symbolic links. Now I don't know whether it really makes a difference since this seems to be Plasma.